js/interface/add-markers.js

```diff
@@ -320,16 +320,31 @@ Fliplet.InteractiveMap.component('add-markers', {
       this.allMarkerStyles.push(newItem)
       this.saveData()
     },
-    deleteMarkerStyle(index) {
+    deleteMarkerStyle(marker) {
```

Comment:
```suggestion
    deleteMarkerStyle(options) {
```

@MaksymShokin Actually use the word `options` to avoid `marker.marker`.

Reply:
If you commit my suggestion, do not forget to still update the code accordingly.

Reply:
My bad. Fixed